ocp_resources/stateful_set.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+from ocp_resources.constants import PROTOCOL_ERROR_EXCEPTION_DICT, TIMEOUT_4MINUTES
+from ocp_resources.logger import get_logger
+from ocp_resources.resource import NamespacedResource
+from ocp_resources.utils import TimeoutSampler
+
+
+LOGGER = get_logger(name=__name__)
+
+
+class StatefulSet(NamespacedResource):
+    api_group = NamespacedResource.ApiGroup.APPS
+
+    def scale_replicas(self, replica_count):
```

Comment:
We have this code in other places, I think we can move it to resource.py and reuse it.